src/v2/BaseLoginRouter.js

```diff
@@ -114,7 +113,6 @@ export default Router.extend({
     // transform response
     const ionResponse = _.compose(
       uiSchemaTransformer,
-      actionsTransformer,
```

Comment:
I don't have the context of what idx.js does. Is the `actionsTransformer` now part of idx.js?

Reply:
yeah, we are now relying on the data that idx-js is giving us.